### PR TITLE
Improve appearance of case title at narrow widths

### PR DESCRIPTION
**Pull request by lowellrex**
Related to goal of making case details page look nice at narrow screen widths, first discussed in #5939.

## 760px wide
| Before | After |
| ------ | ----- |
| image | image |

## 1000px wide
| Before | After |
| ------ | ----- |
| image | image |

### CODE REVIEW

File: client/app/queue/CaseTitle.jsx [hunk 1]
```diff
@@ -7,32 +7,34 @@ import { CATEGORIES } from './constants';
...
-  marginTop: '-2rem'
+  paddingRight: '2.5rem'
 });
 
 const listStyling = css({
-  float: 'left',
   listStyleType: 'none',
-  margin: '-1.5rem 0 0 1rem',
-  padding: 0
+  verticalAlign: 'super',
```
Comment (dannysellers) [inline client/app/queue/CaseTitle.jsx]: Splitting hairs: `super`, or `text-bottom`? Do we want the veteran id etc to align with the baseline of the veteran name title? Or to be vertically offset by a few pixels? I think the `text-bottom` is a bit more readable, but either way's okay with me

`super`
screen shot 2018-06-18 at 1 46 59 pm


`text-bottom`
screen shot 2018-06-18 at 1 47 05 pm

File: client/app/queue/CaseTitle.jsx [hunk 2]
```diff
@@ -7,32 +7,34 @@ import { CATEGORIES } from './constants';
...
+
 const headerStyling = css({
-  float: 'left',
-  listStyleType: 'none',
-  margin: 0,
-  marginTop: '-2rem'
+  paddingRight: '2.5rem'
 });
 
 const listStyling = css({
```
Comment (dannysellers) [inline client/app/queue/CaseTitle.jsx]: I wonder if we should add a couple pixels `margin-top` here. The border divider seems to be touching the text above it (at least when there's a descender in that specific position...)

no margin
screen shot 2018-06-18 at 1 51 27 pm

`margin-top: 5px`
screen shot 2018-06-18 at 1 54 23 pm

File: client/app/queue/CaseTitle.jsx (hunk 1 above)
Comment (lowellrex) [reply client/app/queue/CaseTitle.jsx]: The mocks have the Veteran ID and reader link vertically centered on the Veteran name, but I am happy to change if you and @mkhandekar think that's the way to go. https://app.mural.co/t/workqueue2001/m/workqueue2001/1525112874684/5765d285f568b5f631990e2c8e514a39d9b6b570

File: client/app/queue/CaseTitle.jsx (hunk 2 above)
Comment (lowellrex) [reply client/app/queue/CaseTitle.jsx]: Good call. Changed!

File: client/app/queue/CaseTitle.jsx (hunk 1 above)
Comment (mkhandekar) [reply client/app/queue/CaseTitle.jsx]: I'm 👍 with `text-bottom` !

**Pull request merged**